salesagility/SuiteCRM
# Fix date created

Task: SCRM-575

## Description



When users view emails they cannot see value of the date field.

This change also includes some PHP docs and formatting.

## Motivation and Context

Fixes the date field in the detail views 

## How To Test This

- Repair and rebuild ( Clear tpl cache)
- View emails
- check date created field


## Types of changes


### Final checklist
